guides/common/attributes-base.adoc

```diff
@@ -29,6 +29,22 @@
 :RepoRHEL7ServerSatelliteServerPuppetVersion: rhel-7-server-satellite-6.3-puppet4-rpms
 :RepoRHEL7ServerSatelliteCapsulePuppetVersion: rhel-7-server-satellite-capsule-6.3-puppet4-rpms
 
+//Added by Marie
+// :SatelliteSub: Red Hat Satellite Infrastructure Subscription
+// :RepoRHEL7Server: rhel-7-server-satellite
+// :RepoRHEL8Server: ???
+//:RepoRHEL7ServerSoftwareCollections: rhel-server-rhscl-7-rpms - still valid??
+:RepoRHEL7ServerSatelliteServerProductVersion: rhel-7-server-satellite-capsule-7.0-rpms
+:RepoRHEL8ServerSatelliteServerProductVersion: satellite-7.0-for-rhel-8-x86_64-rpms
+:RepoRHEL7ServerSatelliteCapsuleProductVersion: rhel-7-server-satellite-capsule-7.0-rpms
+:RepoRHEL8ServerSatelliteCapsuleProductVersion: satellite-capsule-7.0-for-rhel-8-x86_64-rpms
+:RepoRHEL7ServerSatelliteMaintenanceProductVersion: rhel-7-server-satellite-maintenance-7.0-rpms
+:RepoRHEL8ServerSatelliteMaintenanceProductVersion: satellite-maintenance-7.0-for-rhel-8-x86_64-rpms
+// Satellite-Tools will become Satellite-Client ???
+// :RepoRHEL7ServerOptional: any equivalent for Sat7???
+// Ansible ??? - not mentioned i nthe doc with repo IDs
```

Comment:
Ansible is used in other guides but defined here with the rest of them. One stop shop!

Reply:
PS: Not reviewing, just giving context.